calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+| `matches_markers`             | `visualization_msgs::msg::MarkerArray` | Publishes matched lidar and radar detection.              |
+| `tracking_markers`            | `visualization_msgs::msg::MarkerArray` | Publishes tracks of reflectors.                           |
+| `text_markers`                | `visualization_msgs::msg::Marker`      | Publishes text markers that show the calibration metrics. |
+| `calibration_metrics`         | `std_msgs::msg::Float32MultiArray`     | Publishes calibration metrics.                            |
+
+### Services
+
+| Name                       | Type                                                  | Description                                                                              |
+| -------------------------- | ----------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `extrinsic_calibration`    | `tier4_calibration_msgs::` `srv::ExtrinsicCalibrator` | Generic calibration service. The call is blocked until the calibration process finishes. |
+| `extract_background_model` | `std_srvs::srv::Empty`                                | Strat to extract the background model from radar and lidar data.                         |
+| `add_lidar_radar_pair`     | `std_srvs::srv::Empty`                                | User is able to click this button to add lidar-radar pair.                               |
+| `delete_lidar_radar_pair`  | `std_srvs::srv::Empty`                                | User is able to click this button to delete the previous lidar-radar pair.               |
+| `send_calibration`         | `std_srvs::srv::Empty`                                | Send the calibration result to the sensor calibration manager.                           |
+
+## Parameters
+
+### Core Parameters
+
+| Name                                        | Type          | Default Value                                           | Description                                                                                                                                       |
+| ------------------------------------------- | ------------- | ------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `radar_parallel_frame`                      | `std::string` | `base_link`                                             | The frame that the radar frame optimizes the transformation to.                                                                                   |
+| `msg_type`                                  | `std::string` | `radar tracks`/`radar scan`                             | The msg type of the input radar message. (Not available yet, currently only support radar tracks)                                                 |
```

Comment:
In the description, you should not use the jargon `msg`

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/3daad85342e764704a624b38d9b123fbd14c0645 (Sorry for accidently mix it with other fix, but obvious to see the difference)